Geometry/TrackerGeometryBuilder/test/ModuleInfo.cc

```diff
@@ -202,22 +166,12 @@ void ModuleInfo::analyze(const edm::Event& iEvent, const edm::EventSetup& iSetup
     }
     Output << std::endl;
     int subdetid = module->geographicalId().subdetId();
-    double volume = gdei->volume() / 1000;  // mm3->cm3
-    double density = gdei->density() / density_units;
```

Comment:
`density_units` is now unused.
Please remove from line 80 of this file: https://github.com/cms-sw/cmssw/pull/31729/files#diff-35e53a427c13b6405f6885a94301343db11c1dae1173319765299c82ac39c5aeR80